### ISSUE TITLE
Add Additional Attributes to Pagination Links

### ISSUE THREAD

**Issue by mtribone** [samvera/hyrax#3138]
### Descriptive summary

In Hyrax 2.1.0 RC3, the pagination links above the footer could use additional attributes for screen reader users. 

Add the following:
- `role="region"` to the element that wraps these pagination links
- `aria-label="pagination links"` to the element that wraps these pagination links
- Screen reader only text before these links which says "Go to page:" or similar.
- `aria-current="page"` to the link that represents the current page. Ensure that this attribute is removed when this link no longer represents the current page.

### Rationale

Because these links are just numbers, it may not be clear to screen reader users that these control pagination.

**Comment by adamjarling** [comment]
PR in Blacklight here:  https://github.com/projectblacklight/blacklight/pull/1970

**Comment by adamjarling** [comment]
Needs to get back ported to Blacklight 6.x, or whatever version Hyrax is currently using.

**Comment by rjkati** [comment]
Closing since this is fixed in Blacklight 7. I did a WAVE report of the page in Hyrax 3.5.0 and the navigation passes in both Firefox and Chrome.
